src/components/dock/dock-button/dock-button.scss

```diff
@@ -0,0 +1,65 @@
+@use '../../../style/mixins';
```

Comment:
> * [ ]  Add labels optionally on mobile devices

@Kiarokh, can you elaborate on what this one means? 🙂

Reply:
Yes. This was a discussion we had with Lucy and decided to leave it for future. 

From accessibility perspective, it's good to have labels together with icons. Which is why many apps have labels on their mobile nav-bars.

**App Store**
![IMG_7966B35DAAC0-1](https://user-images.githubusercontent.com/35954987/180969188-07b7da3a-1040-4cc2-a324-7139d57d11ca.jpeg)
**Linkedin**
![IMG_02AC416698D7-1](https://user-images.githubusercontent.com/35954987/180969242-3d19e1df-dd28-4b76-a27b-2fd16831f64b.jpeg)

However, many other also chose not to, like 
**Instagram**
![IMG_6E5A81968945-1](https://user-images.githubusercontent.com/35954987/180969207-8776be58-944f-44a7-ad88-a4bf37666c04.jpeg)

So we thought we could enable this as an optional feature for the component in future. So if consumer or end-user wants to show labels, a setting can enable them to do so.

Reply:
Ah! Yeah, that makes sense! (Both having it, and not adding it right now 😄)